pkg/util/util.go

```diff
@@ -35,10 +35,16 @@ func SetupFlyteDir() error {
 	if err := os.MkdirAll(f.FilePathJoin(f.UserHomeDir(), ".flyte", "k3s"), os.ModePerm); err != nil {
 		return err
 	}
+
 	// Created a empty file with right permission
-	if err := ioutil.WriteFile(docker.Kubeconfig, []byte(""), os.ModePerm); err != nil {
-		return err
+	if _,err := os.Stat(docker.Kubeconfig); err != nil {
+		if os.IsNotExist(err) {
+			if err := ioutil.WriteFile(docker.Kubeconfig, []byte(""), os.ModePerm); err != nil {
+				return err
+			}
+		}
 	}
```

Comment:
Do we have a case where teardown fails and the kubeconfig exists and prevents further writes to not go through.

We should probably provide some force flag behavior in that case

Reply:
I think we don't need that. In the teardown, we will delete the entire k3s dir

Reply:
There is no harm if kubeconfig exist because in the new sandbox cluster we will mount the new dir.

Reply:
If it fails here https://github.com/flyteorg/flytectl/blob/de0c1c91ad8cd0016331b848403b8937f0875784/cmd/sandbox/teardown.go#L45
the config wont be cleaned up and will impact the furture sandbox writes.

Reply:
No there is no impact on future sandbox write because in new sandbox cluster we will mount the volume and that will replace the old kubeconfig.

Reply:
Ok can you test this please and add the results for it.

Reply:
I manually deleted the sandbox cluster(`docker rm -f flyte-sandbox`) and then created new one and it worked